### PR TITLE
blog(embertimes157): blog post series designer

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-07-24-the-ember-times-issue-157.md [hunk 1]
```diff
@@ -70,14 +71,19 @@ We encourage you to check out Aswathprabhu's [explanation of reactivity](https:/
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## Blog post series of lessons learned working with Ember 📔
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Johan Ronsse (@Wolfr)](https://github.com/Wolfr) wrote a series of blog posts on working with Ember especially coming from a [designer’s point of view](https://mono.company/frontend/working-within-ember-designer-point-of-view/). The [blog](https://mono.company/frontend/getting-to-grips-with-ember/) goes through how the Angle Bracket usage and other features from Ember Octane makes invoking components just like writing HTML which helps designers to be able to focus on the things they care about.
```
Comment (ijlee2) [inline source/2020-07-24-the-ember-times-issue-157.md]: I think we can update the 2nd sentence to be shorter. What do you think about the following, which also removes the word "just (Alex lint)?

```suggestion
[Johan Ronsse (@Wolfr)](https://github.com/Wolfr) wrote a series of blog posts on working with Ember especially coming from a [designer’s point of view](https://mono.company/frontend/working-within-ember-designer-point-of-view/). The blog explains how [angle bracket and other Octane features](https://mono.company/frontend/getting-to-grips-with-ember/) make invoking components feel like writing HTML for designers.
```

File: source/2020-07-24-the-ember-times-issue-157.md [hunk 2]
```diff
@@ -70,14 +71,19 @@ We encourage you to check out Aswathprabhu's [explanation of reactivity](https:/
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In an [example](https://mono.company/frontend/doing-more-with-a-smaller-team-an-ember-octane-example/) showcasing a button component, the blog goes through the abstraction implemented which effectively acts as an enabler for teams by having the code they have to write is much simpler. It’s much harder to make a mistake, because the component has its own simplified API.
```
Comment (ijlee2) [inline source/2020-07-24-the-ember-times-issue-157.md]: I think the 1st sentence is currently hard to parse. What do you think about:
```suggestion
The blog also showcases a [button component](https://mono.company/frontend/doing-more-with-a-smaller-team-an-ember-octane-example/) to illustrate how a good public interface helps end users be effective. The code that they have to write is simpler, which prevents them from introducing errors.
```
?

File: source/2020-07-24-the-ember-times-issue-157.md [hunk 3]
```diff
@@ -70,14 +71,19 @@ We encourage you to check out Aswathprabhu's [explanation of reactivity](https:/
...
+
+Finally, in the [last blog](https://mono.company/frontend/working-within-ember-designer-point-of-view/) post we go through some nifty tools that Ember provides out of the box which makes the complicated easy such as addons and codemods which helps move everyone to the latest Ember release as soon as it becomes available.
```
Comment (ijlee2) [inline source/2020-07-24-the-ember-times-issue-157.md]: I think I'd suggest breaking the sentence into two. Maybe:
```suggestion
Finally, the last blog post covers [nifty tools](https://mono.company/frontend/working-within-ember-designer-point-of-view/) that the Ember ecosystem provides. Addons make the complicated easy and codemods help everyone move to the latest Ember release as soon as it becomes available.
```
?

File: source/2020-07-24-the-ember-times-issue-157.md (hunk 2 above)
Comment (chrisrng) [reply source/2020-07-24-the-ember-times-issue-157.md]: ah yes much better

**Review by ijlee2 — APPROVED** [review]
Chris, much thanks for covering all 3 blog posts! I can imagine that wasn't easy so I appreciate the time that you put into summarizing them. 💯
